features/command_line/order.md

```diff
@@ -13,6 +13,9 @@ order of groups at each level is randomized.
 
 With `rand` you can also specify a seed.
 
+Use `modification_time` to run the most recently modified files first. You can
+combine it with `--next-failure` to find the most recent failing spec.
+
 ## Example usage
 
 The `defined` option is only necessary when you have `--order rand` stored in a
```

Comment:
Unrelated, but this statement seems to be contradictive to the [default ordering setting from the project initializer](https://github.com/rspec/rspec-core/blob/c6315d6e899796d3d0203dc8b656708a3ebca9a1/lib/rspec/core/project_initializer/spec/spec_helper.rb#L92):
```ruby
config.order = :random
```

Reply:
Thats not the default, `defined` is the default, but the convention overrides it, but this sentence is slightly misleading I agree.